CMakeLists.txt

```diff
@@ -166,10 +166,19 @@ else()
   set(SYCLCC_CONFIG_FILE_INSTALL_DIR etc/hipSYCL)
 endif()
 
-if(Boost_FIBER_LIBRARY_DEBUG)
-  get_filename_component(Boost_LIBRARY_DIR ${Boost_FIBER_LIBRARY_DEBUG} DIRECTORY)
+if(NOT Boost_DIR)
```

Comment:
maybe don't check `Boost_DIR` here but instead do `elseif(Boost_FIBER_LIBRARY_RELEASE`) ... else()` with the original if/else?

Reply:
okay